traceapp/app.go

```diff
@@ -85,6 +88,23 @@ func (a *App) serveRoot(w http.ResponseWriter, r *http.Request) error {
 func (a *App) serveTrace(w http.ResponseWriter, r *http.Request) error {
 	v := mux.Vars(r)
 
+	if permalink := r.URL.Query().Get("permalink"); permalink != "" {
+		// If the user specified a permalink, then decode it directly into a
+		// trace structure and place it into storage for viewing.
+		gz, err := gzip.NewReader(base64.NewDecoder(base64.RawURLEncoding, strings.NewReader(permalink)))
```

Comment:
Interesting question.. I checked their sources just now and it does call `QueryUnescape` so it is possible, though when testing locally I did not run into any issues.

I'll have to investigate / confirm, unless maybe @shurcooL would know the answer?